content/hi/nodes.md

```diff
@@ -0,0 +1,18 @@
+---
+title: नोड्स (Nodes) 
+status: Completed
+category: संकल्पना
+exclude_search: true
+---
+
+## यह क्या है
+
+नोड एक कंप्यूटर है जो एक सामान्य कार्य को पूरा करने के लिए अन्य कंप्यूटरों या नोड्स के साथ मिलकर काम करता है। उदाहरण के लिए, अपना लैपटॉप, मॉडेम और प्रिंटर लें। वे सभी आपके वाई-फाई (Wi-Fi) नेटवर्क से जुड़े हुए हैं जो संचार और सहयोग कर रहे हैं, प्रत्येक एक नोड का प्रतिनिधित्व करता है। क्लाउड कंप्यूटिंग में, एक भौतिक कंप्यूटर, एक वर्चुअल कंप्यूटर, (जिसे VM कहा जाता है), या यहां तक ​​कि एक कंटेनर को नोड कहा जा सकता है।
```

Comment:
The sentence should be modified as this, `वे सभी आपके वाई-फाई (Wi-Fi) नेटवर्क से जुड़े रहकर संचार और सहयोग कर रहे हैं और प्रत्येक एक नोड का प्रतिनिधित्व करते है।`